apps/mobile/src/App.tsx

```diff
@@ -116,7 +118,7 @@ export default function App() {
     [colorSchemeLoaded, fontsLoaded]
   );
 
-  if (!fontsLoaded || !colorSchemeLoaded) {
+  if (!fontsLoaded || !colorSchemeLoaded || !introVideoLoaded) {
     return null;
```

Comment:
what happens when we return null?

Reply:
Still showing Splash screen (the Gallery logo)